backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -375,6 +379,66 @@ public string[] GetLayoutNames([CanBeNull] string layoutSetName)
             return layoutNames.ToArray();
         }
 
+        /// <summary>
+        /// Gets the Settings.json for a specific layoutset
+        /// </summary>
+        /// <param name="layoutSetName">The name of the layoutset where the layout belong</param>
+        /// <returns>The content of Settings.json</returns>
+        public async Task<LayoutSettings> GetLayoutSettings(string layoutSetName)
```

Comment:
What method does is special case is not intuitive from the name. It creates a config file if not exists so consider renaming the method name.